pypodio2/areas.py

```diff
@@ -377,17 +377,27 @@ def find_all_for_org(self, org_id):
         """
         return self.transport.GET(url='/org/%s/space/' % org_id)
 
-    def space_members(self, space_id):
+    def space_members(self, space_id, role=None, **kwargs):
         """
         Get a list of active members for a Space.
         https://developers.podio.com/doc/space-members/get-members-of-space-22395
         :param space_id: The unique ID for the space you're asking about
         :type space_id: int
         :type space_id: basestring
+        :type role: None
```

Comment:
`:param role` might be helpful. Is this filtering for members of that specific role?

Reply:
It is, adding.